tasks/pkg-debian.yml

```diff
@@ -2,21 +2,16 @@
 - name: Install apt-transport-https
   apt: name=apt-transport-https state=present
 
-- name: Install ubuntu apt-key server
-  apt_key: id=A2923DFF56EDA6E76E55E492D3A80E30382E94DE keyserver=hkp://keyserver.ubuntu.com:80 state=present
-  when: datadog_apt_key_url_new is not defined
-
-- name: Install Datadog apt-key
-  apt_key: id=A2923DFF56EDA6E76E55E492D3A80E30382E94DE url={{ datadog_apt_key_url_new }} state=present
-  when: datadog_apt_key_url_new is defined
-
-- name: Ensure ubuntu apt-key server is present
-  apt_key: id=C7A7DA52 keyserver=hkp://keyserver.ubuntu.com:80 state=present
-  when: datadog_apt_key_url is not defined
-
-- name: Ensure Datadog apt-key is present
-  apt_key: id=C7A7DA52 url={{ datadog_apt_key_url }} state=present
-  when: datadog_apt_key_url is defined
+- name: Ensure Datadog apt-keys are installed
+  apt_key:
+    id: "{{ item }}"
+    keyserver: "{{ datadog_apt_key_url }}"
```

Comment:
actually, `datadog_apt_key_url` is not a link to a `keyserver` but to an actual direct URL from which the key can be downloaded, so they're necessarily different for the 2 keys.

So instead of `with_items` you could use `with_dict` to be able to specify both keys' fingerprint and url